### PR TITLE
Sean Translation test

### PR DESCRIPTION
**Pull request by Boran**

### CODE REVIEW

File: frontend/messages/fr.json [hunk 1]
```diff
@@ -38,8 +38,8 @@
   "locationSounding": "Émagramme",
   "locationSoundingLegend": "Émagramme pour cette heure et ce lieu",
   "period24HoursBefore": "24 heures plus tôt",
-  "periodPrevious": "Période prévue précédente",
-  "periodNext": "Période prévue suivante",
+  "periodPrevious": "Période de prévision antérieure",
```
Comment (julienrf) [inline frontend/messages/fr.json:41]:
```suggestion
  "periodPrevious": "Période de prévision précédente",
```

**Review by julienrf — COMMENTED** [review]
Thank you, I just have a minor suggestion regarding the french text.

**Comment by Boran** [comment]
Julien, here is the merge of 
- Italian machine generated textes
- Corrected by Alvise
- and 2-3 French corrections

File: frontend/messages/fr.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
   "summaryTotalCloudCover": "Couverture nuageuse",
   "summaryRainfall": "Précipitations",
   "summaryCumuliDepth": "Épaisseur des cumulus",
-  "summarySoaringLayerDepth": "Couche convective",
+  "summarySoaringLayerDepth": "Epaisseur couche convective",
```
Comment (julienrf) [inline frontend/messages/fr.json:71]: This message appears in a place where space is expensive:

image

I would recommend keeping “Couche convective” or maybe abbreviating to “Épaisseur couche conv.”

Comment (julienrf) [reply frontend/messages/fr.json:71]:
```suggestion
  "summarySoaringLayerDepth": "Épaisseur couche conv.",
```

Preview:

image

**Review by julienrf — COMMENTED** [review]
Thank you for fixing the typos and thank you for the Italian translation!

**Review by julienrf — APPROVED** [review]

**Pull request merged**